ci/docker/runtime_functions.sh

```diff
@@ -382,7 +381,6 @@ unittest_ubuntu_python3_cpu() {
     #export MXNET_MKLDNN_DEBUG=1  # Ignored if not present
     export MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
     nosetests-3.4 --verbose tests/python/unittest
-    nosetests-3.4 --verbose tests/python/quantization
```

Comment:
Please leave this in. It's supposed to be here, right @reminisce ?

Reply:
the same here.

Reply:
@zheng-da We cannot pass this test_quantization.py when building without mkldnn.

Reply:
@zhang-da @reminisce The difficulty is we need a way to separate the naive CPU quantization and MKLDNN quantization test if we want to share the same test_quantization.py (it's our aim to share code as @reminisce mentioned) since they are both use CPU context, I figured out a way to set a environment variable USE_MKLDNN in test_quantization_mkldnn.py and check this in test_quantization.py, for naive CPU path USE_MKLDNN will not be set so will go to naive path. We have added quantization test for CPU path back in new diff. Please help to review and check if there is any better way to do this. Thanks.

Reply:
Okay.
Ideally, we'd have a C-API function that provides information about the backend. E.g. 'Is_mkl_present'. This is a hack and I'm fine with it, but ideally we'd have a reliable information source.